system/jlib/jlog.cpp

```diff
@@ -2176,7 +2310,11 @@ MODULE_INIT(INIT_PRIORITY_JLOG)
     thePassAllFilter = new PassAllLogMsgFilter();
     thePassLocalFilter = new PassLocalLogMsgFilter();
     thePassNoneFilter = new PassNoneLogMsgFilter();
+#ifndef _CONTAINERIZED
+    theStderrHandler = new HandleLogMsgHandlerJSON(stderr, MSGFIELD_STANDARD);
```

Comment:
a) This test is the wrong way around.
b) It should is if (isContainerized()) instead of a #ifdef so that the syntax of both branches are checked.
c) Is this a good idea?  It will increase the size of the logs, and probably make them harder for humans to scan.

Reply:
> a) This test is the wrong way around. b) It should is if (isContainerized()) instead of a #ifdef so that the syntax of both branches are checked. c) Is this a good idea? It will increase the size of the logs, and probably make them harder for humans to scan.

Obviously the JSON format has higher overhead than space delimited flat format, the concept is that JSON formatted data can be trivially interpreted and routed/filtered by automated post processors at an early stage, thereby providing huge storage and processing improvement potential. If the benefits of JSON are not desirable, in certain situations, it should definitely be optionally switched off.

Given the availability of log processing solutions (with log query languages and user interfaces, and many other powerful tools) the use-case for human scan-ability is greatly diminished, compared to the legacy (antiquated?) methodology of manually scouring huge flat files. In production, raw trace logs should not be considered human scannable, rather machine readable and consumable.